src/Tools/dotnet-monitor/Startup.cs

```diff
@@ -109,12 +104,11 @@ public static void Configure(IApplicationBuilder app, IWebHostEnvironment env, I
             {
                 builder.MapControllers();
 
-                // Use a redirect to the Swagger UI if the request hits the default endpoint.
-                // This means that the swagger endpoint can have a permanent address, even if we decide to host
-                // something else at the root later.
-                builder.MapGet("/", (HttpResponse response) =>
+                builder.MapGet("/", (HttpResponse response, ISwaggerProvider provider) =>
```

Comment:
Double checking -- this route doesn't require authentication to access, correct?

Reply:
Correct